particle-accelerator/src/renderer/views/LoginView.jsx

```diff
@@ -0,0 +1,31 @@
+import { useContext } from 'react'
+import LoginContext from '../utilities/LoginContext'
+import TokenContext from '../utilities/TokenContext'
+import LoginStatus from '../components/LoginStatus'
+import LoginForm from '../components/LoginForm'
+
+/**
+ * LoginView: React Component that provides a login form for a user to login
+ * through to access the ActivatorView.
+ */
+export default function LoginView() {
+  const { setToken, token } = useContext(TokenContext)
+  const { setLoginState, loginState } = useContext(LoginContext)
+
+  return (
+    <>
+      <h2>Particle Accelerator</h2>
+      {/* eslint-disable-next-line react/jsx-no-constructed-context-values */}
+      <LoginContext.Provider value={{ loginState, setLoginState }}>
+        {/* eslint-disable-next-line react/jsx-no-constructed-context-values */}
+        <TokenContext.Provider value={{ token, setToken }}>
+          <LoginStatus />
+        </TokenContext.Provider>
+        {/* eslint-disable-next-line react/jsx-no-constructed-context-values */}
```

Comment:
Rather than doing this ignore three times, you could just go it once at the top of the file. It would make this easier to read.

Reply:
Done!